src/js/views/admin/sidebar/clinician/clinician-sidebar_views.js

```diff
@@ -1,27 +1,54 @@
+import _ from 'underscore';
+import Backbone from 'backbone';
 import hbs from 'handlebars-inline-precompile';
+import Radio from 'backbone.radio';
 import { View } from 'marionette';
 
 import 'sass/modules/buttons.scss';
 import 'sass/modules/forms.scss';
 import 'sass/modules/sidebar.scss';
 
+import intl from 'js/i18n';
+
+import { animSidebar } from 'js/anim';
+
 import { GroupsComponent, RoleComponent, AccessComponent } from 'js/views/admin/shared/clinicians_components';
+import Optionlist from 'js/components/optionlist';
 
 import ClinicianSidebarTemplate from './clinician-sidebar.hbs';
 
 import './clinician-sidebar.scss';
 
+const i18n = intl.admin.sidebar.clinician.clinicianSidebarViews;
+
 const NameView = View.extend({
   className: 'pos--relative',
-  template: hbs`<input class="input-primary w-100 js-input" placeholder="{{ @intl.admin.sidebar.clinician.clinicianSidebarViews.nameView.placeholder }}" value="{{ name }}" />`,
+  template: hbs`
+    <input class="input-primary w-100 js-input" placeholder="{{ @intl.admin.sidebar.clinician.clinicianSidebarViews.nameView.placeholder }}" value="{{ name }}" {{#unless canEdit}}disabled{{/unless}} />{{~ remove_whitespace ~}}
+    {{#unless canEdit}}<span class="clinician-sidebar__locked-icon">{{far "lock"}}</span>{{/unless}}
+    {{#if error}}<div class="form-error u-margin--b-4">{{ error }}</div>{{/if}}
+  `,
+  templateContext() {
+    return {
+      canEdit: this.model.isEditable(),
+      error: this.error,
+    };
+  },
   ui: {
     input: '.js-input',
   },
   events: {
     'input @ui.input': 'onInput',
   },
+  initialize({ error }) {
+    this.error = error;
+  },
   onInput(evt) {
     this.model.set('name', evt.target.value);
+    if (this.error) {
```

Comment:
No super thrilled with this, but it might be what's needed to achieve the clearing of the error. We don't want to call `showName` in the layout view because that will remove focus on the element every time the user types.

Reply:
you can check program, but I think we leave up the error until the next submission.  If that's the case we probably just need `getOption` in the `templateContext`